core/neat/mixins/_grid-push.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+/// Push or pull a Neat grid column.
+///
+/// @argument {number (unitless)} $push [1]
+///
+/// @argument {map} $grid [$neat-default-grid]
+///   The grid to be used to generate the column. By default, the global
+///   `$neat-default-grid` will be used.
+///
+/// @example scss
+///   .element {
+///     @include grid-push(3);
+///   }
+///
+/// @example css
+///   .element {
+///     margin-left: calc((100% - (260px)) * 0.25 + 80px);
+///   }
+
+@mixin grid-push($push: false, $grid: $neat-default-grid) {
+  $_grid-columns: map-get($grid, columns);
+  $_grid-gutter: map-get($grid, gutter);
+
+  @if $push {
```

Comment:
Same general comment as above, the `0`/`null`/`false` option exists if we need to *reset* a previously defined `grid-push` to its default state.

Side note here, the `>= 1` option would not work because a user can also "pull" an object by passing in negative values. In this case a `=! 0` would be more appropriate.